tests/test_ec2/test_route_tables.py

```diff
@@ -147,6 +147,22 @@ def test_route_tables_filters_standard():
         [route["GatewayId"] == igw.id for table in resp for route in table["Routes"]]
     )
 
+    # Filter by route destination CIDR block
+    resp = client.describe_route_tables(
+        Filters=[
+            {"Name": "route.destination-cidr-block", "Values": ["10.0.0.4/24"]},
+        ]
+    )["RouteTables"]
+    resp.should.have.length_of(1)
+    resp[0]["RouteTableId"].should.equal(route_table2.id)
+    assert any(
```

Comment:
```suggestion
    all_routes = itertools.chain(..)
    assert len(all_routes) >= 1
    assert all([route["DestinationCidrBlock"] == "10.0.0.4/24" for route in all_routes])
```
Just a suggestion. I don't recall out of the top of my head how to flatten the response using itertools, but this would make the test agnostic against other resources. We only need to know that we have at least one route, and that all returned routes match our specified CidrBlock

Reply:
> all returned routes match our specified CidrBlock

I don't think that's the behaviour we want. As per [AWS](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeRouteTables.html):

> `route.destination-cidr-block` - The IPv4 CIDR range specified in a route in the table.

i.e. not all routes will have the specified CidrBlock